src/test/java/org/openrewrite/maven/RewriteRunIT.java

```diff
@@ -18,142 +18,157 @@
-                .isSuccessful()
-                .out()
-                .warn()
-                .noneSatisfy(line -> assertThat(line).contains("Unable to parse checkstyle configuration"));
+          .isSuccessful()
+          .out()
+          .warn()
+          .noneSatisfy(line -> assertThat(line).contains("Unable to parse checkstyle configuration"));
     }
 
     @MavenTest
     void recipe_project(MavenExecutionResult result) {
         assertThat(result)
-                .isFailure()
-                .out()
-                .error()
-                .anySatisfy(line -> assertThat(line).contains("/sample/ThrowingRecipe.java", "This recipe throws an exception"));
+          .isFailure()
+          .out()
+          .error()
+          .anySatisfy(line -> assertThat(line).contains(slashForSystem("/sample/ThrowingRecipe.java"), "This recipe throws an exception"));
     }
 
     @MavenTest
     void cloud_suitability_project(MavenExecutionResult result) {
         assertThat(result)
-                .isSuccessful()
-                .out()
-                .warn()
-                .anySatisfy(line -> assertThat(line).contains("some.jks"));
+          .isSuccessful()
+          .out()
+          .warn()
+          .anySatisfy(line -> assertThat(line).contains("some.jks"));
     }
 
     @MavenTest
     @SystemProperties({
-            @SystemProperty(value = "rewrite.activeRecipes", content = "org.openrewrite.maven.RemovePlugin"),
-            @SystemProperty(value = "rewrite.options", content = "groupId=org.openrewrite.maven,artifactId=rewrite-maven-plugin")
+      @SystemProperty(value = "rewrite.activeRecipes", content = "org.openrewrite.maven.RemovePlugin"),
+      @SystemProperty(value = "rewrite.options", content = "groupId=org.openrewrite.maven,artifactId=rewrite-maven-plugin")
     })
     void command_line_options(MavenExecutionResult result) {
         assertThat(result).isSuccessful().out().error().isEmpty();
         assertThat(result).isSuccessful().out().warn()
-                .contains("Changes have been made to target/maven-it/org/openrewrite/maven/RewriteRunIT/command_line_options/project/pom.xml by:")
-                .contains("    org.openrewrite.maven.RemovePlugin: {groupId=org.openrewrite.maven, artifactId=rewrite-maven-plugin}");
+          .contains(slashForSystem("Changes have been made to target/maven-it/org/openrewrite/maven/RewriteRunIT/command_line_options/project/pom.xml by:"))
+          .contains("    org.openrewrite.maven.RemovePlugin: {groupId=org.openrewrite.maven, artifactId=rewrite-maven-plugin}");
         assertThat(result.getMavenProjectResult().getModel().getBuild()).isNull();
     }
 
+    @SystemProperties({
+      @SystemProperty(value = "rewrite.activeRecipes", content = "org.openrewrite.java.AddCommentToMethod"),
+      @SystemProperty(value = "rewrite.options", content = "comment='{\\\"test\\\":{\\\"some\\\":\\\"yeah\\\"}}',methodPattern=sample.SomeClass doTheThing(..)")
```

Comment:
This test passes if I instead use 
```suggestion
      @SystemProperty(value = "rewrite.options", content = "comment='{\"test\":{\"some\":\"yeah\"}}',methodPattern=sample.SomeClass doTheThing(..)")
```

Reply:
So on Windows it does this if I change the test to that:
```
java.lang.AssertionError: 
Expecting ArrayList:
  ["This build will only read from the build cache, since the clean lifecycle is not part of the build invocation.",
    "bootstrap class path not set in conjunction with -source 8",
    "source value 8 is obsolete and will be removed in a future release",
    "target value 8 is obsolete and will be removed in a future release",
    "To suppress warnings about obsolete options, use -Xlint:-options.",
    "Changes have been made to target\maven-it\org\openrewrite\maven\RewriteRunIT\command_line_options_json\project\src\main\java\sample\SomeClass.java by:",
    "    org.openrewrite.java.AddCommentToMethod: {comment='{test:{some:yeah}}', methodPattern=sample.SomeClass doTheThing(..)}",
    "Please review and commit the results.",
    "Estimate time saved: 5m"]
to contain:
  ["    org.openrewrite.java.AddCommentToMethod: {comment='{"test":{"some":"yeah"}}', methodPattern=sample.SomeClass doTheThing(..)}"]
but could not find the following element(s):
  ["    org.openrewrite.java.AddCommentToMethod: {comment='{"test":{"some":"yeah"}}', methodPattern=sample.SomeClass doTheThing(..)}"]


	at org.openrewrite.maven.RewriteRunIT.command_line_options_json(RewriteRunIT.java:120)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.ForkJoinTask.doExec$$$capture(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
```